Assets/Scripts/CharacterBehaviour.cs

```diff
@@ -0,0 +1,75 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class CharacterBehaviour : MonoBehaviour
+{
+    //MOUSE CONTROL
+    public float lookSpeed, lookXLimit;
+    float rotationX = 0;
```

Comment:
im unsure why the access modifiers are different here, why cant rotationX explicitly be `public` or even `private`?